src/hooks/profile/useUpdateAccountsProfile.ts

```diff
@@ -0,0 +1,31 @@
+import { useQuery } from '@apollo/client';
+import GetProfileForAddresses from 'services/graphql/queries/GetProfileForAddresses';
+import { useStoredAccountsAddresses } from '@recoil/accounts';
+import { convertGraphQLProfile } from 'lib/GraphQLUtils';
+import { DesmosProfile } from 'types/desmos';
+import { useStoreProfiles } from '@recoil/profiles';
+
+/**
+ * Hook that updates all the profiles associated to the accounts that the user
+ * have imported in the application.
+ */
+export const useUpdateAccountsProfile = () => {
```

Comment:
I think it might be better to name this `useUpdateAccountsProfiles` (with trailing `s`) since this updates multiple profiles